pkg/util/version/version.go

```diff
@@ -126,6 +126,16 @@ func (v *Version) BuildMetadata() string {
 	return v.buildMetadata
 }
 
+// PreRelease returns the prerelease metadata, if v is a Semantic Version, or ""
+func (v *Version) PreRelease() string {
+	return v.preRelease
+}
+
+// Components returns the version number components
+func (v *Version) Components() []uint {
```

Comment:
What I thought of adding yet here is `Major() uint`, `Minor() uint` and `Patch() uint` which would work like this:
```
func (v *Version) Patch() uint {
    if len(v.components) < 3 {
        return 0
    }
    return v.components[2]
}
```
etc.

Looks much cleaner in the code...